packages/pancake-uikit/src/components/Card/index.stories.tsx

```diff
@@ -57,8 +58,38 @@ export const Default: React.FC = () => {
 };
 
 export const CardHeader: React.FC = () => {
+  const theme = useTheme();
+  // This is example how to make card header "overlap" the border.
+  // Seems to be easiest solution that works on all screens and does not rely on absolute positioning trickery
+  const headerHeight = "60px";
+  const customHeadingColor = "#7645D9";
+  const gradientStopPoint = `calc(${headerHeight} + 1px)`;
+  const borderBackground = `linear-gradient(${customHeadingColor} ${gradientStopPoint}, ${theme.colors.cardBorder} ${gradientStopPoint})`;
+
+  // Gradient overlap is also possible, just put the "dividing" gradient first and after that the header gradient
+  const gradientBorderColor = `linear-gradient(transparent ${gradientStopPoint}, ${theme.colors.cardBorder} ${gradientStopPoint}), ${theme.colors.gradients.cardHeader}`;
   return (
     <div style={{ padding: "32px", width: "500px" }}>
+      <Row>
+        <Card borderBackground={borderBackground}>
```

Comment:
All this stuff is just a trick to make the effect of card header overlapping the 1px border without using some ugly extra divs and absolute positioning. Right now needed only for Predictions.
![Screenshot 2021-08-08 at 20 21 07](https://user-images.githubusercontent.com/81824236/128640371-5eebb351-65ac-46be-983a-d11d13750f51.png)